docker-compose.yml

```diff
@@ -366,11 +366,10 @@ services:
                 ip route add default via 40.40.40.1 dev eth0.40 table 1001
                 sleep 5 && \
                 ping -c 3 6.6.6.6 -I 40.40.40.40 && ping -c 3 40.40.40.41 && \
-                ip link add link eth1 name eth1.50 type vlan id 50 && \
-                ip link set eth1.50 up && \
-                ip addr add 50.50.50.50/24 dev eth1.50 && \
+                ip addr add 50.50.50.50/24 dev eth1 && \
                 ip rule add from 50.50.50.50 lookup 1002
-                ip route add default via 50.50.50.1 dev eth1.50 table 1002
+                ip route add default via 50.50.50.1 dev eth1 table 1002
```

Comment:
Please use 50.50.50.5 instead of 50.50.50.1

Reply:
done https://github.com/opiproject/opi-evpn-bridge/pull/156/commits/ef000a4d003a030cc33cf1cfc1ac831a44a16f0f